### DISCUSSION TITLE
Invalid `prisma.link.findMany()` invocation

### DISCUSSION THREAD

**Discussion by 28development** [prisma/prisma#9565] [Q&A] [answered]
I went through [this](https://www.prisma.io/blog/fullstack-nextjs-graphql-prisma-2-fwpc6ds155) tutorial: 

And somehow I get an error in my apollo query

The error:

`Invalid `prisma.link.findMany()` invocation: Argument take must not be null. Please use undefined instead`

I think this is due to my ```graphql/resolvers.ts``` file which looks like this:

```
export const resolvers = {
    Query: {
        links: (_parent, _args, ctx) => {
            return ctx.prisma.link.findMany()
        },
    },
}
```

Not sure what I am doing wrong.

**Comment by 28development** [comment, marked as answer]
Nevermind I forgot to set the linksFirst argument to a value in the apollo ui, setting a value fixed this :)

This can be closed

**Comment by mittalyashu** [comment]
I am getting similar while writing integration tests.

Running the actual endpoint from a client it works fine but throws `Invalid `prisma.users.findMany()` invocation in` this error while running tests.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
